CMakeLists.txt

```diff
@@ -18,8 +18,12 @@ function(packageProject)
     ${ARGN}
   )
 
+  if("${PROJECT_INCLUDE_DESTINATION}" STREQUAL "include/${PROJECT_NAME}-${PROJECT_VERSION}")
+    set(VERSION_POSTFIX -${PROJECT_VERSION})
+  endif()
```

Comment:
I don't like the implicit activation of the feature. What if I a user would like a different include path but still would want a version suffix to be appended? 

How about adding an additional parameter `NO_VERSION_SUFFIX` instead, that users could specify in the call to disable the feature?

Reply:
That is also an option, but the include path has to be consistent with the project settings!
```cmake
# either set(INCLUDE_DESTINATION include/${PROJECT_NAME}-${PROJECT_VERSION})
# or any other ...
set(INCLUDE_DESTINATION include/${PROJECT_NAME})
target_include_directories(
    Greeter PUBLIC $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
                   $<INSTALL_INTERFACE:${INCLUDE_DESTINATION}>
)
```

Reply:
How should I prevent conflicting parameter settings?

Reply:
any suggestions for testing?